### DISCUSSION TITLE
How to develop andorid app with kotlin-language-server

### DISCUSSION THREAD

**Discussion by Decodetalkers** [fwcd/kotlin-language-server#355] [Q&A] [answered]
It always cannot find android and andoridx.. Yesterday, when I run it with android studio, it work, however, in the afternoon, It cannot find these again.. I feel very confused..I use neovim with kotlin-language-server

**Comment by Decodetalkers** [comment, marked as answer]
> It always cannot find android and andoridx.. Yesterday, when I run it with android studio, it work, however, in the afternoon, It cannot find these again.. I feel very confused..I use neovim with kotlin-language-server

Ok ,gradle wrapper fix it

**Reply by chaitanyabsprip** [reply]
Gradle wrapper fixed it how, could you please share how you got it to work?

**Reply by Decodetalkers** [reply]
> Gradle wrapper fixed it how, could you please share how you got it to work?

Emmm. Finally make it works should be open a android studio, let it compile the program, then the lsp will work
